changelog.md

```diff
@@ -1,5 +1,80 @@
 ## 15.x
 
+### 15.0-beta2 (2025-03-15)
+
+- Feature: Snow-covered rocks are now visible (#13627)
+- Feature: Place rocks on "too steep" tiles when fixing slopes (#13462)
+- Feature: Prevent towns from upgrading individually-placed houses (#13270)
+- Feature: [Win32] 2D map scrolling (#13172)
+- Feature: NewGRF Badges (#13073)
+- Feature: Convert 32bpp-only sprites to 8bpp when needed (#11602)
```

Comment:
I'm not sure this is a `Feature`, because who chooses to use the 8bpp blitter? (Suggestion will need to be moved to the proper order if accepted)

```suggestion
- Add: Convert 32bpp-only sprites to 8bpp when needed (#11602)
```

Reply:
This one is relevant to NewGRF authors, not players. But we don't have a separate changelog for that.